src/GregClient/Requests/PackageVersionUpload.cs

```diff
@@ -44,8 +44,8 @@ internal override void Build(ref RestRequest request)
 
             ((PackageVersionUploadRequestBody)this.RequestBody).file_hash = Convert.ToBase64String(zipHash);
 
-            // pass pkg_header as parameter
```

Comment:
do we have any idea why this was done originally?

Reply:
Cannot be sure, but probably unintentional side effect of using the auth api